ppr-ui/src/store/mutations/mutations-model.ts

```diff
@@ -372,6 +372,10 @@ export const mutateMhrHomeOwners = (state: StateIF, owners: Array<MhrRegistratio
   state.stateModel.mhrRegistration.owners = owners
 }
 
+export const mutateMhrRegistrationDocumentId = (state: StateIF, value) => {
+  state.stateModel.mhrRegistration.documentId = value
+}
+
```

Comment:
Likewise here, we know `value` is of type `string` here i think

Reply:
Added return type, thanks!